main.go

```diff
@@ -247,6 +247,35 @@ func FS(useLocal bool) http.FileSystem {
 	return _esc_static
 }
 
+// FSByte returns the content of an embedded asset as a byte slice.
+func FSByte(useLocal bool, path string) ([]byte, error) {
+	r, err := FS(useLocal).Open(path)
+	if err != nil {
+		return nil, err
+	}
+	return ioutil.ReadAll(r)
+}
+
+// FSMustByte is the same as FSByte but panics on error.
+func FSMustByte(useLocal bool, path string) []byte {
+	if b, err := FSByte(useLocal, path); err != nil {
+		panic(err)
+	} else {
```

Comment:
No need for the else: https://github.com/golang/go/wiki/CodeReviewComments#indent-error-flow